testnets/celestiatestnet3/chain.json

```diff
@@ -22,29 +22,29 @@
   },
   "codebase": {
     "git_repo": "https://github.com/celestiaorg/celestia-app",
-    "recommended_version": "v0.11.1",
+    "recommended_version": "v1.0.0-rc2",
     "compatible_versions": [
-      "v0.12.0"
+      "v1.0.0-rc2"
     ],
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/celestiaorg/networks/master/mocha/genesis.json"
     },
     "versions": [
       {
         "name": "v0.11.1",
```

Comment:
the name would stay as v0.11.1?
Would it be possible to retain the old version in the versions array?

Reply:
thanks for reviewing! hm, since it is a new version `mocha-2` instead of `mocha`, the compatibility is not the same between the two and `mocha` will be deprecated

Reply:
oops. it definitely is in this case 😄 thank you for catching that! editing now